web/solutii/cristian_danila/rest_client/Solution.java

```diff
@@ -0,0 +1,105 @@
+import org.json.JSONObject;
+
+import javax.print.DocFlavor;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class NetClientGet {
+
+    public static void main(String[] args) {
+
+        try {
+
+            // do a get request for all the users
+            String[] resources = new String[] {"posts", "comments", "albums", "photos", "todos", "users"};
+
+            for (String resource : resources) {
+
+                URL url = new URL("http://jsonplaceholder.typicode.com/" + resource);
+                HttpURLConnection conn = (HttpURLConnection) url.openConnection();
+                conn.setRequestMethod("GET");
+                conn.setRequestProperty("Accept", "application/json");
+
+                if (conn.getResponseCode() != 200) {
+                    throw new RuntimeException("Failed : HTTP error code : "
```

Comment:
At the first status cod that is not `200` this app stops o.O ? Why ? There is no response to the user and not future connections.